Client/Frontend/Theme/LegacyThemeManager/ThemedWidgets.swift

```diff
@@ -93,6 +93,45 @@ class ThemedTableViewCell: UITableViewCell, ReusableCell, ThemeApplicable {
     func configure(viewModel: ThemedTableViewCellViewModel) {
         self.viewModel = viewModel
     }
+
+    func restoreToInitialState() {
```

Comment:
This should normally be handled by using [`prepareForReuse`](https://developer.apple.com/documentation/uikit/uitableviewcell/1623223-prepareforreuse) instead. I don't think we need to reset all the things as well in there (like font is normally not needed?)

Reply:
Ok perfect indeed I didn't think to put it there. Anyway, the font is needed to be restored to standard because there are cells with different font like the user cell, and without that I saw some cells having bold font that shouldn't be having.